Source/WebCore/rendering/updating/RenderTreeUpdaterViewTransition.h

```diff
@@ -41,7 +41,7 @@ class RenderTreeUpdater::ViewTransition {
 
     void updatePseudoElementTree(RenderElement&, StyleDifference minimalStyleDifference);
 private:
-    void buildPseudoElementGroup(const AtomString&, RenderElement&, RenderObject* = nullptr);
+    void buildPseudoElementGroup(RenderBox& viewTransitionRoot, const AtomString&, RenderElement&, RenderObject* = nullptr);
```

Comment:
shouldn't this be a block and not a box?

Reply:
I guess it can (even RenderBlockFlow, need to downcast either way), since we unconditionally use that for `::view-transition`. I think that's somewhat incorrect, since the author could have provided a `display: inline` style, but that's tangential.